### DISCUSSION TITLE
Vector to Raster (from Preprocessing menu)

### DISCUSSION THREAD

**Discussion by Almeida100** [semiautomaticgit/SemiAutomaticClassificationPlugin#338] [General]
As is known, the QGis "Align rasters..." tool has a bug, which transfers the NODATA value to the DATA area.
Fortunately, SCP has a very useful tool, in the "Preprocessing -> Vector to raster" menu, which allows transferring data from vector to raster, aligning this raster with a reference raster and also assigning the NODATA value that the user desires!
However, the result is always a raster with the data type of "Int32 - Thirty two bit signed integer".
My question: will it not be possible in a future version of SCP to allow the user to choose the data type, for example, between the current one and "Float32 - Thirty two bit floating point"?

**Comment by semiautomaticgit** [comment]
Thank you @Almeida100 , yes I think it is possible to add this feature.
